database/src/main/kotlin/no/nav/su/se/bakover/database/DatabaseBuilder.kt

```diff
@@ -19,24 +33,40 @@ object DatabaseBuilder {
         Flyway(abstractDatasource.getDatasource(Postgres.Role.Admin), databaseName).migrate()
 
         val userDatastore = abstractDatasource.getDatasource(Postgres.Role.User)
-        val objectRepo = DatabaseRepo(userDatastore)
         return DatabaseRepos(
-            objectRepo = objectRepo,
-            avstemmingRepo = AvstemmingPostgresRepo(userDatastore)
+            avstemmingRepo = AvstemmingPostgresRepo(userDatastore),
+            utbetalingRepo = UtbetalingPostgresRepo(userDatastore),
+            oppdragRepo = OppdragPostgresRepo(userDatastore),
+            søknadRepo = SøknadPostgresRepo(userDatastore),
+            behandlingRepo = BehandlingPostgresRepo(userDatastore),
+            hendelsesloggRepo = HendelsesloggPostgresRepo(userDatastore),
+            beregningRepo = BeregningPostgresRepo(userDatastore),
+            sakRepo = SakPostgresRepo(userDatastore)
         )
     }
 
     fun build(embeddedDatasource: DataSource): DatabaseRepos {
         Flyway(embeddedDatasource, "postgres").migrate()
-        val objectRepo = DatabaseRepo(embeddedDatasource)
         return DatabaseRepos(
-            objectRepo = objectRepo,
-            avstemmingRepo = AvstemmingPostgresRepo(embeddedDatasource)
+            avstemmingRepo = AvstemmingPostgresRepo(embeddedDatasource),
+            utbetalingRepo = UtbetalingPostgresRepo(embeddedDatasource),
+            oppdragRepo = OppdragPostgresRepo(embeddedDatasource),
+            søknadRepo = SøknadPostgresRepo(embeddedDatasource),
+            behandlingRepo = BehandlingPostgresRepo(embeddedDatasource),
+            hendelsesloggRepo = HendelsesloggPostgresRepo(embeddedDatasource),
+            beregningRepo = BeregningPostgresRepo(embeddedDatasource),
+            sakRepo = SakPostgresRepo(embeddedDatasource)
         )
     }
 }
 
 data class DatabaseRepos(
-    val objectRepo: ObjectRepo,
-    val avstemmingRepo: AvstemmingRepo
+    val avstemmingRepo: AvstemmingRepo,
+    val utbetalingRepo: UtbetalingRepo,
+    val oppdragRepo: OppdragRepo,
+    val søknadRepo: SøknadRepo,
+    val behandlingRepo: BehandlingRepo,
+    val hendelsesloggRepo: HendelsesloggRepo,
+    val beregningRepo: BeregningRepo,
+    val sakRepo: SakRepo
```

Comment:
Siden dette allerede kalles `DatabaseRepos`, kanskje vi kunne ha forenklet til
```suggestion
    val avstemming: AvstemmingRepo,
    val utbetaling: UtbetalingRepo,
    val oppdrag: OppdragRepo,
    val søknad: SøknadRepo,
    val behandling: BehandlingRepo,
    val hendelseslogg: HendelsesloggRepo,
    val beregning: BeregningRepo,
    val sak: SakRepo
```